src/components/Filters/FilterRow/FilterRow.tsx

```diff
@@ -166,10 +174,9 @@ const renderComponent = (Component, value, onChange) => {
   }
   // Component with props
   if (typeof Component === 'object' && has('props', Component)) {
-    return renderComponentWithProps(Component, value, onChange);
+    return renderComponentWithProps(Component, value, onChange, onError);
   }
-
-  return <Component value={value} onChange={onChange} />;
+  return <Component value={value} onChange={onChange} onError={onError} />;
```

Comment:
I think we don't need to pass onError to component without additional properties because all validations should be defined in component props object. Or I am missing some use case?